x-pack/plugins/observability/public/components/app/section/index.tsx

```diff
@@ -31,9 +47,21 @@ export function SectionContainer({ title, appLink, children, hasError }: Props)
         id={title}
         buttonContentClassName="accordion-button"
         buttonContent={
-          <EuiTitle size="xs">
-            <h5>{title}</h5>
-          </EuiTitle>
+          <>
+            <EuiFlexGroup gutterSize="s" alignItems="center" responsive={false}>
+              <EuiFlexItem grow={false}>
+                <EuiTitle size="xs">
+                  <h5>{title}</h5>
+                </EuiTitle>
+              </EuiFlexItem>
+
+              {showExperimentalBadge && (
+                <EuiFlexItem grow={false}>
+                  <ExperimentalBadge />
```

Comment:
@estermv Oh now I realized that you're importing the same technical preview badge from the Alerts view, so that makes sense to re-use. The result looks the same and the status is kept in sync 👍